src/app/components/candidate-card-expanded/candidate-card-expanded.component.html

```diff
@@ -69,12 +69,12 @@ <h2>{{ candidate['description'] }}</h2>
     <div class="p-col-12">
       <div class="title">
         <p>
-          Donors by Group
+          Donations by Group
           <sup>
             <i
               class="helper-tip fa fa-question-circle"
               aria-hidden="true"
-              matTooltip="Description goes here."
+              matTooltip="Total contributions from five largest groups of campaign donors.  Groups are determined by the industry segment of each donors’ employer."
```

Comment:
I think it sounds better "... the five largest groups...".

Reply:
@nancyheiss Done